lib/rototiller/utilities/env_var_checker.rb

```diff
@@ -24,7 +24,7 @@ def check_env_vars
     end
     abort_message = 'Aborting Rake:'
     required_vars.each{ |v| abort_message << red_text("\nThe environment variable #{v.name} is required. #{v.message}")}
-    abort(abort_message) unless required_vars.empty?
+    raise abort_message unless required_vars.empty?
```

Comment:
this should probably use an exception class like 
http://ruby-doc.org/core-2.2.0/ArgumentError.html

it should probably also be handled (read: logged, styled) in an exception handler upstream.  but i'm willing to let this one in for now because we don't have a bunch of these messages, yet.